docs/create-handlers.md

```diff
@@ -3,14 +3,326 @@ id: create-handlers
+Game.SteamID = "1234"; | Here the SteamID of the game. You can find it on https://steamdb.info/ . Add it only if the game is available on Steam.
+Game.LauncherTitle = ""; | The name of the launcher's window title. Some games need to go through a launcher to open. This is needed or else the application will lose the game's window.
+Game.LauncherExe = "gamelauncher.exe";  | If the game needs to go through a launcher before opening. Nucleus will use this exe to start the game and it will follow Game.ExecutableName process for resizing and positioning. You can also write it like this: Game.LauncherExe = "gamelauncher.exe|NucleusDefined"; for let the User select the launcher exe the very first time (useful if is not located in the game folder).
+Game.LauncherExeIgnoreFileCheck = true;	 | Do not check if Launcher Exe exists in game folder | you will need to provide a relative filepath from game root folder
+
+***Files copies lines***
+
+Usually, these are enough:
+Line | Description
+---- | -----------
+Game.SymlinkExe = false; | False = this will copy the game exe. Enable = will symlink the game exe.
+Game.SymlinkGame = true; | False = this will not copy the game in Nucleus\content folder. Enable = this will symlink the game in Nucleus\content folder
+
+***Others that could help:***
+Line | Description
+---- | -----------
+Game.HardlinkGame = true;          | **Nucleus-Coop must be on the same game drive**. Different type of link files. Hardlink files instead of Symlink (or hard copy) | Directories will still be symlinked but files will be hardlinked
+
+***For saves and testing:***
+Line | Description
+---- | -----------
+Game.KeepSymLinkOnExit = true; | False = after Nucleus is closed, game files inside Nucleus\content will be deleted. Enable = Nucleus will keep the files inside Nucleus\content even after is closed. Useful if the game store saves it in the game-folder.
+
+## Nucleus-Coop Environment
+
+This function will provide a different game's Document path for each player.
+
+When a player is initialized, Nucleus-Coop will edit the Windows registry lines to make the game use a different path for search/create Documents files.
+You can find the registry lines inside this path:
+**HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders**
+
+Then, you will be able to access in the script context part by using:
+- Context.EnvironmentPlayer (specific player Environment path) or  Context.EnvironmentRoot (Nucleus-Coop Environment Folder root) when using the UserProfile lines.
+- Context.DocumentsPlayer (specific player Documents path) or Context.DocumentsRoot (Nucleus-Coop Documents Folder root) when using the Documents lines.
+
+Line | Description
+---- | -----------
+Game.UseNucleusEnvironment = true; | Use custom environment variables for games that use them, replaces some common paths (e.g. AppData) with C:\Users\<your username>\NucleusCoop
```

Comment:
This line causes an error because `<your username>` isn't escaped:
![image](https://user-images.githubusercontent.com/43886029/128763184-81455193-0520-496d-9978-6d432dc514db.png)

You can fix it by escaping the `<` and `>` signs:  `Game.UseNucleusEnvironment = true; | Use custom environment variables for games that use them, replaces some common paths (e.g. AppData) with C:\Users\\<your username\>\NucleusCoop`